src/patternfly/components/Accordion/accordion.scss

```diff
@@ -80,7 +83,8 @@
       }
 
       .pf-c-accordion__toggle-icon {
-        transform: rotate(90deg);
+        transition: var(--pf-c-accordion__toggle-icon--expanded--Transition);
```

Comment:
```suggestion
        transition: var(--pf-c-accordion__toggle-icon--m-expanded--Transition);
```

Reply:
You also want to move the `transition` to `.pf-c-accordion__toggle-icon` outside of the context of `.pf-m-expanded`, otherwise the transition will only happen when the element already has `.pf-m-expanded` (it will transition "collapsed", but won't transition "expanded")

Reply:
also just left a comment in https://github.com/patternfly/patternfly-next/issues/1938, but let's change this var and the one below to `--pf-c-accordion__toggle--m-expanded__toggle-icon--[Transition/Transform]`

Reply:
@mcoker why did you add in the extra toggle into the name?

Reply:
It's just following the selector names, and `toggle-icon` is the BEM element name for the toggle icon. Since the selector order is `.pf-c-accordion__toggle.pf-m-expanded` -> `.pf-c-accordion__toggle-icon`, the var would be `--pf-c-accordion__toggle--m-expanded__toggle-icon`